addons/purchase/models/purchase_order.py

```diff
@@ -1064,19 +1058,7 @@ def _get_product_price_and_data(self, product):
                 price=seller.price_discounted,
                 min_qty=seller.min_qty,
             )
-        # Check if the product uses some packaging.
-        packaging = self.env['product.packaging'].search(
-            [('product_id', '=', product.id), ('purchase', '=', True)], limit=1
-        )
-        if packaging:
-            qty = packaging.product_uom_id._compute_quantity(packaging.qty, product.uom_po_id)
-            product_infos.update(
-                packaging={
-                    'id': packaging.id,
-                    'name': packaging.display_name,
-                    'qty': qty,
-                }
-            )
```

Comment:
You remove the feature of packaging in catalog. It should also be replaced with UoM base on pricelist.

Reply:
![image](https://github.com/user-attachments/assets/cd267d2b-7dd1-4b17-98b9-8584da85f65e)